hydra/app/views/catalog/_document.html.erb

```diff
@@ -1,25 +1,25 @@
-<% if params[:q].present? 
+<% if params[:q].present?
      @page_title = t('blacklight.search.page_title.title', :constraints => render_search_to_page_title(params), :application_name => application_name)
    else
 	 @page_title = "Browsing #{application_name}"
-   end 
+   end
 %>
 
 <% # container for a single doc -%>
 <div class="document <%= render_document_class document %>" itemscope itemtype="<%= document.itemtype %>">
 	<div class="document-wrap">
-        <% identifier = document[:id] %>
```

Comment:
See renaming variable; we have a case where we've renamed identifier to id and id to identifier.  That adds a lot of chatter to the PR.

Reply:
acknowledged that it adds chatter, unfortunately that's just how it will be unless we want to keep it misnamed